### DISCUSSION TITLE
Error: Text content does not match server-rendered HTML

### DISCUSSION THREAD

**Discussion by HypeillingerChillinger** [vinissimus/next-translate#1212] [General]
Using version 2.6.2 of next-translate
and NextJs with app router

The problem occurs with or without 'use client'
```Error: Text content does not match server-rendered HTML```

```typescript
// some other useState s
const {t, lang} = useTranslation('navbar');
return (
    <div>{t('Hello')}</div>
);
```

What am I doing wrong?

**Comment by wesnolte** [comment]
Try the canary release: https://github.com/aralroca/next-translate/releases
